x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/job/RollupJobTask.java

```diff
@@ -240,6 +262,8 @@ public synchronized void start(ActionListener<StartRollupJobAction.Response> lis
                             listener.onResponse(new StartRollupJobAction.Response(true));
                         },
                         (exc) -> {
+                            // We were unable to update the persistent status, so we need to shutdown the indexer too.
+                            indexer.stop();
```

Comment:
This is unrelated to the bugfix, but looked like a bug in it's own right so I fixed here.  We shouldn't let the indexer keep running if we weren't able to persist the state... could lead to a strange situation where we are persisted as `stopped` but indexing a bunch of data.